components/user_settings/display/user_settings_theme/custom_theme_chooser.jsx

```diff
@@ -376,18 +393,34 @@ export default class CustomThemeChooser extends React.Component {
                 <label className='custom-label'>
                     <FormattedMessage
                         id='user.settings.custom_theme.copyPaste'
-                        defaultMessage='Copy and paste to share theme colors:'
+                        defaultMessage='Copy to share or paste theme colors here:'
                     />
                 </label>
                 <textarea
                     ref='textarea'
                     className='form-control'
                     id='pasteBox'
                     value={this.state.copyTheme}
+                    onCopy={this.showCopySuccess}
                     onPaste={this.pasteBoxChange}
                     onChange={this.onChangeHandle}
                     onClick={this.selectTheme}
                 />
+                <div className='margin-top x2'>
+                    <button
+                        className='btn btn-link copy-theme-button'
+                        onClick={this.copyTheme}
+                    >
+                        {'Copy Theme Colors'}
```

Comment:
I think we should use FormattedMessage here, otherwise it won't be translated

Reply:
done